docs/auth/oauth2-proxy/provider.md

```diff
@@ -0,0 +1,105 @@
+---
+id: provider
+title: OAuth 2 Proxy Provider
+sidebar_label: OAuth 2 Custom Proxy
+description: Adding OAuth2Proxy as an authentication provider in Backstage
+---
+
+The Backstage `auth-backend` package comes with an `oauth2proxy` authentication
```

Comment:
haha but the package name is actually `@backstage/plugin-auth-backend` ... :D

Reply:
Just tried to keep it in sync with the other docs 🙈 but the full name is fine for me 😄

Reply:
Updated 👍